### PR TITLE
Adding Omar M to README.MD

### PR DESCRIPTION
**Pull request by omeneses**
Adding Omar M to README.md file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,4 +19,4 @@ All team members should do the following:
 ## Team members
 | Name | Profile            |
 | ---- | ------------------ |
-| Name |[Username](github.link) |
\ No newline at end of file
+| Omar M |[omeneses](github.link) |
```
Comment (mfbarrera) [inline README.md]: falta el link de Git.

**Review by mfbarrera — DISMISSED** [review]

File: README.md (hunk 1 above)
Comment (omeneses) [reply README.md]: > falta el link de Git.

Done!!!  Thanks dude!!!

**Comment by jsebax** [comment]
Did you agree with your team in one naming convention for the branch? I see there are two PRs with different naming convention

**Comment by omeneses** [comment]
> Did you agree with your team in one naming convention for the branch? I see there are two PRs with different naming convention

Yeah mate, we agreed to use : feature/name

**Review by caritonaranjo — DISMISSED** [review]

**Review by MarianaLoaiza — DISMISSED** [review]

**Review by caritonaranjo — APPROVED** [review]

**Review by mfbarrera — APPROVED** [review]

**Pull request merged**